### PR TITLE
Feature/wabo dossiers

### PR DESCRIPTION
**Pull request by yelsherbini**

### CODE REVIEW

File: src/bouwdossiers/batch.py [hunk 1]
```diff
@@ -90,7 +90,101 @@ def openbaar_to_access(openbaar):
     return models.ACCESS_RESTRICTED
 
 
-def add_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
+def add_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
+    """
+    Add wabo dossier to the the bouwdossier model. Structure of import is
+    almost identical to pre_wabo xml to avoid confusion since only a few
+    mappings are different and new fields are added.
```
Comment (kramer65) [inline src/bouwdossiers/batch.py:97]: Please add a short description of what "wabo" and "pre-wabo" means. That way future readers will have some sort of idea what they're looking at.

File: src/bouwdossiers/batch.py [hunk 2]
```diff
@@ -90,7 +90,101 @@ def openbaar_to_access(openbaar):
...
+        titel = ''
+        log.warning(f"Missing titel for Wabo dossier {dossiernr} in {file_path}")
+
+    datering = x_dossier.get('begindatum')
+    dossier_type = x_dossier.get('omschrijving')
+
+    bouwdossier = models.BouwDossier(
+        importfile=import_file,
+        dossiernr=dossiernr,
+        stadsdeel=stadsdeel.upper(),  # Make it upper to follow the same format as pre_wabo dossiers
```
Comment (kramer65) [inline src/bouwdossiers/batch.py:113]: Maybe it's a good idea to do a check on whether the stadsdeel code which is provided here is correct and part of the possibilities/choices.

File: src/bouwdossiers/batch.py [hunk 3]
```diff
@@ -90,7 +90,101 @@ def openbaar_to_access(openbaar):
...
+            document_type=x_document.get('document_type')
+        )
+
+        documenten.append(document)
+
+    models.Document.objects.bulk_create(documenten)
+
+    return count, total_count
+
+def add_pre_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
```
Comment (kramer65) [inline src/bouwdossiers/batch.py:187]: Also add a comment here about what wabo is

File: src/bouwdossiers/tests/test_api.py [hunk 4]
```diff
@@ -192,3 +193,33 @@ def test_invalid_dossiernr(self):
...
+        adres.huisnummer_letter = 'A'
+        adres.huisnummer_toevoeging = 'B'
+        adres.save()
+
+        url = '/stadsarchief/bouwdossier/AA12345/'
+        response = self.client.get(url)
+        documents = response.data.get('documenten')
+        adressen = response.data['adressen']
+        self.assertEqual(response.data['olo_liaan_nummer'], 12345)
+        self.assertEqual(response.data.get('wabo_bron'), None)  # Bron ignored
```
Comment (kramer65) [inline src/bouwdossiers/tests/test_api.py:219]: Why is the bron ignored?

File: src/bouwdossiers/batch.py (hunk 1 above)
Comment (yelsherbini) [reply src/bouwdossiers/batch.py:97]: Good point, I think this is more suited in the README however.

**Pull request merged**